### PR TITLE
Add ability to set keyboard shorcuts

### PR DESCRIPTION
**Pull request by liljaylj**
... using browser.commands API.

Additionally, added new "Clear selection" command. It is useful if you prefer keyboard-driven navigation.

### CODE REVIEW

File: background/background.js [hunk 1]
```diff
@@ -49,3 +60,11 @@ function buildMenu(menuGroupDict, disabledItemSet) {
         first = false;
     }
 }
+
+function hydrateCommandDescriptions(groups) {
+    for (const [group, commands] of Object.entries(groups)) {
+        for (const [name, title] of Object.entries(commands)) {
```
Comment (l10nelw) [inline background/background.js:66]: Again, reserving plural names for arrays. Most objects functioning as 'dictionaries' (like 'maps', except not using the Map type) in this repo are given a `Dict` suffix.

```suggestion
function hydrateCommandDescriptions(groupDict) {
    for (const [group, commandDict] of Object.entries(groupDict)) {
        for (const [name, title] of Object.entries(commandDict)) {
```

Also please move this function above `function buildMenu()` to be consistent with your other command-related code (command first then menu).

File: background/get.js [hunk 2]
```diff
@@ -129,3 +129,8 @@ async function getTabsAccessedOnDay(offset) {
     };
     return (await queryTabs()).filter(isLastAccessedAtDate);
 }
+
+/* Clear selection */
+export async function clearSelection(targetTab) {
+  return [ targetTab ];
```
Comment (l10nelw) [inline background/get.js:135]: This is not the expectation if you invoke it via context menu. This command should return the active tab.

File: menudata.js [hunk 3]
```diff
@@ -21,5 +21,8 @@ export default {
         past24Hours:            'Accessed in the Past 24 H&ours',
         today:                  'Accessed &Today',
         yesterday:              'Accessed &Yesterday',
+    },
+    'Clear': {
+        clearSelection:         'Clear selectio&n',
```
Comment (l10nelw) [inline menudata.js:26]:
```suggestion
    'Other': {
        none:         'Clear selection (&Z)',
```
Note that the commands are named to reflect their outcomes. So I prefer to call it `none`.

I don't feel `N` is right for this command. `C` is already taken. `Z` seems appropriate, it reminds us of 'undo'. Your hydrate function will need to remove `(&?)` as well.

**Review by l10nelw — CHANGES_REQUESTED** [review]
This is great idea @liljaylj. Do you normally navigate tabs via keyboard? How do you do it?

**Comment by ces42** [comment]
What is the holdup on this? I would love to see this merged.

**Comment by l10nelw** [comment]
@ces42 @liljaylj merged.

**Pull request merged**